terraform/validators.tf

```diff
@@ -175,6 +174,8 @@ variable "log_to_file" {
 
 locals {
   validator_command = var.log_to_file ? jsonencode(["bash", "-c", "/docker-run-dynamic.sh >> /opt/libra/data/libra.log 2>&1"]) : ""
+  aws_elasticsearch_host = var.enable_logstash ? join(",", aws_elasticsearch_domain.logging.*.endpoint) : ""
+  logstash_config  = "input { file { path => '/opt/libra/data/libra.log'\\n}}\\n output {  amazon_es { \\nhosts => ['https://${local.aws_elasticsearch_host}']\\nregion => 'us-west-2'\\nindex => 'validator-logs-%%{+YYYY.MM.dd}'\\n}}"
```

Comment:
A `template_file` would be nicer, but okay for now.

Reply:
Yeah, formatting this is a bit tricky, cause we need to render this in another template. I will see if we can make this looks better later.